tfjs-backend-webgpu/src/kernels/matmul_packed_vec4_webgpu.ts

```diff
@@ -162,14 +163,157 @@ export function makeMatMulVectorVec4Source(): string {
   `;
 }
 
+export function makeMatMulPackedVec4SourceWgsl(
+    varSnippet: string, workPerThread: number[],
+    workGroupSize: [number, number, number]): string {
+  const rowPerThread = workPerThread[1];
+  const colPerThread = workPerThread[0];  // only support colPerThread = 4
+  const tileBOuter = workGroupSize[0] * colPerThread;
+  const tileInfo = {
+    RowPerThread: workPerThread[1],
+    ColPerThread: workPerThread[0],
+    TileAOuter: workGroupSize[1] * rowPerThread,
```

Comment:
Can we use `TileAOuter: workGroupSize[1] * RowPerThread`?

Reply:
done